extensions/ql-vscode/test/pure-tests/databases/db-config-validator.test.ts

```diff
@@ -25,9 +25,10 @@ describe('db config validation', async () => {
 
     const validationOutput = configValidator.validate(dbConfig);
 
-    expect(validationOutput).to.have.length(2);
+    expect(validationOutput).to.have.length(3);
```

Comment:
Shall we check the last validation error too? Or put this back to 2 and add local stuff to dbConfig on L13